### PR TITLE
OCPQE-24437: Support workflow-launching a hosted cluster on AWS using QE cluster profile

### PR DESCRIPTION
**Pull request by fxierh**
A template job has been added to `infra-periodics.yaml` to enable the use of the `aws-qe` cluster profile within workflows-config.yaml. 

This allows us to operate within our own budget and use our usual `BASE_DOMAIN` and `HYPERSHIFT_BASE_DOMAIN`.

Note: Similar changes were made for Azure in PR #54888, and Azure-hosted clusters can now be successfully workflow-launched using the expected cluster profile.

### CODE REVIEW

File: core-services/ci-chat-bot/workflows-config.yaml
Comment (bradmwilliams) [inline core-services/ci-chat-bot/workflows-config.yaml:263]: Before this change can merge, you'll need to add `aws-qe` as a support platform in the clusterbot:
https://github.com/openshift/ci-chat-bot/blob/5305b6ceebe072fb2918b487b8747e109702e510/pkg/manager/prow.go#L56
or the workflow will almost immediately fail with an unsupported platform error.

**Review by bradmwilliams — COMMENTED** [review]
/hold
Needs changes to the clusterbot before this workflow will work.

**Review by bradmwilliams — APPROVED** [review]

File: core-services/ci-chat-bot/workflows-config.yaml
Comment (fxierh) [reply core-services/ci-chat-bot/workflows-config.yaml:263]: Thanks for the comment

Comment (fxierh) [reply core-services/ci-chat-bot/workflows-config.yaml:263]: For future reference: 
This works because workflow-launch uses different validation logic. As [discussed](https://redhat-internal.slack.com/archives/CNHC2DK2M/p1722609379193539) in the channel, we'll keep the current SupportedPlatforms unchanged to avoid exposing the -qe platforms to the public.

**Pull request merged**
